model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -879,6 +879,29 @@ message StandardCoders {
     // Components: None
     // Experimental.
     ROW = 13 [(beam_urn) = "beam:coder:row:v1"];
+
+    // Encodes a use key and a shard id which is an opaque byte string.
+    //
+    // The encoding for a sharded key consists of the length prefixed shard id
+    // and the encoded user key in the following order:
+    //
+    //     varInt(len(shard id))
```

Comment:
Instead of referencing varInt(len(shard id)) shard id we should just say that the shard id is encoded using `beam:coder:bytes:v1`

Other coders descriptions should be more like row where we refer to other well known encodings.